--- a/dwm.c	2015-11-09 01:39:37.000000000 +0300
+++ b/dwm.c	2016-01-18 11:12:48.000000000 +0300
@@ -194,6 +194,7 @@
 static void resizemouse(const Arg *arg);
 static void restack(Monitor *m);
 static void run(void);
+static void runAutostart(void);
 static void scan(void);
 static int sendevent(Client *c, Atom proto);
 static void sendmon(Client *c, Monitor *m);
@@ -1398,6 +1399,12 @@
 }
 
 void
+runAutostart(void) {
+    system("cd ~/.dwm; ./autostart_blocking.sh");
+    system("cd ~/.dwm; ./autostart.sh &");
+}
+
+void
 scan(void)
 {
 	unsigned int i, num;
@@ -2134,6 +2141,7 @@
 	checkotherwm();
 	setup();
 	scan();
+    runAutostart();
 	run();
 	cleanup();
 	XCloseDisplay(dpy);
